packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -822,6 +826,9 @@ export default defineComponent({
       }
 
       return this.$gettext('Personal')
+    },
+    getArePathsDisplayed(resource) {
```

Comment:
```suggestion
    arePathsDisplayed(resource) {
```

Reply:
I named it like that because a) there is already a property called `arePathsDisplayed` and b) I kinda like the "get"-prefix to indicate we're dealing with a method here.